doc/nrf/ug_nrf70_developing_powersave.rst

```diff
@@ -0,0 +1,324 @@
+.. _ug_nrf70_developing_powersave:
+
+Operating in power save modes
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports multiple power save modes, enabling the device to minimize power consumption by shutting down most of the digital logic and RF circuits.
+
+nRF70 Series device states
+**************************
+
+The power save state of the nRF70 Series device is described through a combination of the physical Power state of the logic or circuits and the logical Functional state as observed by the access point (AP).
+
+Power state
+===========
+
+The nRF70 Series device can operate in either of the following two power states:
+
+* **Active:** The device is **ON** constantly so that it can receive and transmit the data.
+* **Sleep:** The device is **OFF** to majority of the blocks that can not receive and transmit the data.
+  In this state, the device consumes low power (~15 µA).
+
+Functional state
+================
+
+The state information of the device at the access point database can be either Active or Power Save.
+The device is in the Active state after the connection to a network.
+
+The state change information must be given to an access point by a successful frame exchange.
+The nRF70 Series device can operate in either of the following two functional states:
+
+* **Active:** The device is in the Active state permanently.
+  If the data exists, the access point can schedule transmission of the data to the device immediately.
+* **Power Save (PS):** The device is in the Sleep state most of the time, but can also be woken up to transmit and receive data.
+  In this state, the access point buffers the required frames intended for the device.
+  The device can make a transition between the Active and Sleep states according to the IEEE 802.11 power management standards.
```

Comment:
I think this is partly related to my comment on line 41. I think we are using the terms "power management" and "power save" to mean the same thing,  and if so would like us to converge on "power save". For line 39, it could possibly be written as "IEEE802.11 power save protocol". But I think rules would also work, since the sentence is saying that a device is free to switch in and out of power save, but must follow the rules defined in the power save protocol within the IEEE802.11 standard.